plugins/ltac2/tac2stdlib.ml

```diff
@@ -618,3 +618,12 @@ let () =
   define "evarconv_unify"
     (transparent_state @-> constr @-> constr @-> tac unit)
     Tac2tactics.evarconv_unify
+
+let () =
+  define "congruence"
+  (option int @-> option (list constr) @-> tac unit)
+  Tac2tactics.congruence
+let () =
+  define "simple_congruence"
+  (option int @-> option (list constr) @-> tac unit)
+  Tac2tactics.simple_congruence
```

Comment:
Files should end with a single newline character but this (and others in this PR) ends with a non newline character.

Reply:
```suggestion
  Tac2tactics.simple_congruence

```

Reply:
Is there a way to run the CI linter localy? I search for a lint target in the Makefile.ci or a ci-lint.sh in dev/ci but haven't found something like this.

Reply:
https://github.com/coq/coq/blob/master/dev/lint-repository.sh

Reply:
Thank you